### PR TITLE
Plasmid new application page

### PR DESCRIPTION
**Pull request by dev-rajneeshkumar**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: 
- https://main--moleculardevices--hlxsites.hlx.page/applications/plasmid/aav-plasmids
- https://main--moleculardevices--hlxsites.hlx.page/applications/plasmid/plasmids-3d-culture-organoid-screening
-  https://main--moleculardevices--hlxsites.hlx.page/applications/coronavirus-research
- After:
-  https://plasmid--moleculardevices--hlxsites.hlx.page/applications/plasmid/aav-plasmids
 -  https://plasmid--moleculardevices--hlxsites.hlx.page/applications/plasmid/plasmids-3d-culture-organoid-screening
 -  https://plasmid--moleculardevices--hlxsites.hlx.page/applications/coronavirus-research

### CODE REVIEW

File: templates/blog/blog.css
[the remark this answers is not in the document]
Comment (dev-rajneeshkumar) [reply templates/blog/blog.css:1]: This is to remove twitter feeds are broken and add new static html for tweets. Got merged in same branch as other PR was not reviewed yet

**Comment by dev-rajneeshkumar** [comment]
Hi @alexcarol  @alinarublea  @andreituicu  can someone please review it? Seems to be a conflict in accordion.js

File: blocks/accordion/accordion.js [hunk 1]
```diff
@@ -56,30 +57,11 @@ async function renderContent(container, content, isBlockFaq) {
       });
     }
   });
-  if (isBlockFaq) {
```
Comment (alexcarol) [inline blocks/accordion/accordion.js:59]: Are the changes in this file intended or is it a merge/rebase accident?

Comment (dev-rajneeshkumar) [reply blocks/accordion/accordion.js:59]: @alexcarol Yes changes required for a request, actually there was conflict over this  so can you please confirm if this block was merged and required?

**Comment by dev-rajneeshkumar** [comment]
Closing this due to multiple conflicts and opened fresh one https://github.com/hlxsites/moleculardevices/pull/1198

**Pull request closed without merging**